server/gc/src/main/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogs.java

```diff
@@ -55,12 +56,16 @@
 import org.apache.zookeeper.KeeperException;
 
 import com.google.common.net.HostAndPort;
+import java.util.concurrent.TimeUnit;
+import org.apache.accumulo.core.conf.Property;
 
 public class GarbageCollectWriteAheadLogs {
   private static final Logger log = Logger.getLogger(GarbageCollectWriteAheadLogs.class);
 
   private final Instance instance;
   private final VolumeManager fs;
+  private final Map<HostAndPort,Long> firstSeenDead = new HashMap<HostAndPort,Long>();
```

Comment:
No, I don't actually think it's necessary (I haven't looked closely enough), but, at first glance, it looks like it be necessary. I believe the caller of GCWALs does so in such a way that it does not have any race conditions.

Re-worded, I think this is probably fine as-is, but some documentation saying "This is not threadsafe, but the caller is not invoking it in a concurrent manner" would be good for future devs/readers.